Lombiq.NodeJs.Extensions/package.json

```diff
@@ -36,23 +46,18 @@
     "clean": "run-p --print-label --continue-on-error \"clean:*\"",
     "watch": "run-p --print-label --continue-on-error \"watch:*\"",
     "--": "-------------------------------------------------- STYLES -------------------------------------------------",
-    "build:styles":
-      "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=${SCSS_SOURCE:-Assets/Styles} SCSS_TARGET=${SCSS_TARGET:-wwwroot/css} pnpm _build:styles",
+    "build:styles": "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=${SCSS_SOURCE:-Assets/Styles} SCSS_TARGET=${SCSS_TARGET:-wwwroot/css} pnpm _build:styles",
     "build:styles:args": "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=$npm_config_source SCSS_TARGET=$npm_config_target pnpm _build:styles",
     "_build:styles": "run-p --print-label --continue-on-error _compile:styles _lint:styles",
     "lint:styles": "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=Assets/Styles pnpm _lint:styles",
     "lint:styles:args": "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=$npm_config_source pnpm _lint:styles",
-    "_lint:styles": "stylelint \"../../$SCSS_SOURCE/**/*.scss\" || AREA=styles STEP=linting pnpm error",
-    "compile:styles":
-      "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=${SCSS_SOURCE:-Assets/Styles} SCSS_TARGET=${SCSS_TARGET:-wwwroot/css} pnpm _compile:styles",
+    "_lint:styles": "stylelint --allow-empty-input \"../../$SCSS_SOURCE/**/*.scss\" || AREA=styles STEP=linting pnpm error",
```

Comment:
When there aren't any styles to compile. However it's no good because then it just fails at `_sass:styles` instead so it's not much of an improvement.

Reply:
Yeah, I have to fix that soon.

Reply:
Let's keep it though.